pkgs/development/python-modules/misaki/default.nix

```diff
@@ -0,0 +1,117 @@
+{
+  addict,
+  buildPythonPackage,
+  espeak-ng,
+  fetchFromGitHub,
+  fetchpatch2,
+  fugashi,
+  hatchling,
+  jaconv,
+  jamo,
+  jieba,
+  lib,
+  nltk,
+  num2words,
+  ordered-set,
+  phonemizer,
+  pip,
+  pypinyin,
+  pythonOlder,
+  regex,
+  replaceVars,
+  spacy-curated-transformers,
+  spacy,
+  stdenv,
+  torch,
+  transformers,
+  unidic,
+}:
+
+buildPythonPackage {
+  pname = "misaki";
+  version = "0-unstable-2025-06-16";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "hexgrad";
+    repo = "misaki";
+    rev = "49ddead831fc1a46beab3f9cb6064d93fd3f6347";
+    hash = "sha256-XkmCG0Vkw/iyLNmwhaW5gp6sg8ieHFeK6fBJ3Jb+J8Q=";
+  };
+
+  patches = [
+    (fetchpatch2 {
+      name = "pr82-remove-pip-dependency.patch";
+      url = "https://github.com/hexgrad/misaki/commit/7d2fdff9fe046dd52b638bbb1b243fb789f4cb31.patch";
```

Comment:
Have you read the docs? Please append ?full_index=1.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches

> GitHub supports appending ?full_index=1 to the URL